pandas/util/_decorators.py

```diff
@@ -272,18 +278,14 @@ def deprecate_nonkeyword_arguments(
 
     Parameters
     ----------
-    version : str, optional
-        The version in which positional arguments will become
-        keyword-only. If None, then the warning message won't
-        specify any particular version.
-
+    klass : Warning, optional
```

Comment:
is this optional? `None` has been removed from the annotations for `klass`?

Reply:
Thanks!